modules/ROOT/pages/subscriptions/events.adoc

```diff
@@ -747,7 +747,7 @@ subscription {
                 }
            }
         }
-    }
+    }As an example, consider these type definitions:
```

Comment:
this is inside a code listing - intended?

Reply:
Nice catch!